template/client/store/actions.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as mutationTypes from '~/store/types'
+import type { StateType } from '~/types'
+import type { ActionContext } from 'vuex'
+import comments from '~/logics/api/comments'
+
+export default {
```

Comment:
One minor feature that could be improved:

```
const actions = {...}

export default actions
```

This way we can:
1. run additional code before exporting it
2. have extra name that allows us to understand what we actually export

The same applies for: `getters`, `mutations`, `actions`.